data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -2606,6 +2607,61 @@
 },
 {
   "type" : "recipe",
+  "result": "chem_saltpetre",
+  "byproducts": { "id" : "ammonia", "amount" : 1 },
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_CHEMICALS",
+  "skill_used": "cooking",
+  "difficulty": 5,
+  "time": 25000,
+  "book_learn": [[ "textbook_anarch", 7 ] , [ "recipe_labchem", 5 ], ["textbook_chemistry", 6]],
+  "qualities" : [
+      { "id":"BOIL","level":2 },
+      { "id":"CHEM","level":2 }
+  ],
+  "tools": [
+    [
+      [ "fire", -1 ],
+      [ "toolset", 25 ],
+      [ "hotplate", 25 ]
+    ]
+  ],
+  "components": [
+    [ [ "chem_ammonium_nitrate", 25 ] ],
+    [ [ "lye_powder", 145 ] ]
```

Comment:
Round numbers (to multiple of 25, in case of big numbers) are preferred. Exact chemical composition isn't really important here, as all the components are approximate anyway.